lib/tasks/import_data_to_bigquery.rake

```diff
@@ -0,0 +1,24 @@
+DEFAULT_BATCH_SIZE = 1
+SLEEP_TIME = 2
+
+namespace :bigquery do
+  desc 'Import model data to BigQuery as import_entity type events'
+  task :import_entity_events, %i[model_name sleep_time start_at_id] => :environment do |_, args|
+    abort('You must specify the model whose data you want to importa to BigQuery e.g. rake bigquery:import_entity_events[ApplicationChoice]') if args[:model_name].blank?
+
+    model_name = args[:model_name]
+    sleep_time = args[:sleep_time].to_i || SLEEP_TIME
```

Comment:
I think if `args[:sleep_time]` is nil, calling `to_i` on it will return 0 ... and `SLEEP_TIME` will never be used.

```suggestion
    sleep_time = args.fetch(:sleep_time, SLEEP_TIME).to_i
```